src/test/java/guitests/guihandles/BikeListPanelHandle.java

```diff
@@ -18,6 +18,10 @@ protected String getItemClassName() {
         return Bike.class.getSimpleName();
     }
 
+    protected String getCardPaneId() {
```

Comment:
Id is quite ambiguous as it can mean the ID (eg 4,5 or 6 or LOAN_ID) of the CardPane. Should we use getCardPaneCssId instead?